include/modem/modem_info.h

```diff
@@ -32,7 +32,22 @@ extern "C" {
 #define MODEM_INFO_JSON_STRING_SIZE 512
 
 /** RSRP offset value. */
-#define MODEM_INFO_RSRP_OFFSET_VAL 141
+#define MODEM_INFO_RSRP_OFFSET_VAL 140
+
+/** RSRQ offset value. */
+#define MODEM_INFO_RSRQ_OFFSET_VAL 19.5
+
+/** RSRQ scale value. */
+#define MODEM_INFO_RSRQ_SCALE_VAL 0.5
+
+/** Modem returns RSRP and RSRQ as index values which require
+ * a conversion to dBm and dB respectively. See modem AT
+ * command reference guide for more information.
+ */
+#define MODEM_INFO_RSRP_ADJ(rsrp)  ((rsrp) - MODEM_INFO_RSRP_OFFSET_VAL)
```

Comment:
perhaps something like `RSRP_IDX_TO_DBM` and `RSRQ_IDX_TO_DB` would be better than `_ADJ`?

Reply:
Yes, good suggestion.  I had modeled these macros after the RSRP_ADJ and RSRQ_ADJ macros that were inside a couple of .c files.